app/forms/funding/bursary_form.rb

```diff
@@ -4,14 +4,26 @@ module Funding
   class BursaryForm < TraineeForm
     FIELDS = %i[
       applying_for_bursary
+      bursary_tier
     ].freeze
 
     attr_accessor(*FIELDS)
 
+    before_validation :set_applying_for_bursary
+
     validates :applying_for_bursary, inclusion: { in: [true, false] }
+    validates :bursary_tier, inclusion: { in: Trainee.bursary_tiers.keys }, allow_blank: true
+
+    def applying_for_bursary=(value)
+      @applying_for_bursary = ActiveModel::Type::Boolean.new.cast(value)
+    end
 
   private
 
+    def set_applying_for_bursary
+      self.applying_for_bursary = true if bursary_tier.present?
```

Comment:
```suggestion
      self.applying_for_bursary = bursary_tier.present?
```

Reply:
Hmm, I think this might introduce a subtle bug.
For the non-tiered bursary form, `applying_for_bursary` should not depend on the value of `bursary_tier`, since that would be explicitly chosen by the user.